common/app/views/fragments/analytics/google.scala.html

```diff
@@ -109,6 +106,7 @@
                         dimension12: referrerVars.value.path
                     });
                 }
+                window.sessionStorage.removeItem(NG_STORAGE_KEY);
```

Comment:
Added to remove the session storage item that was previously removed in the Omniture scripts